### DISCUSSION TITLE
增加一个新网盘--比特球网盘

### DISCUSSION THREAD

**Discussion by youtehub** [AlistGo/alist#2589] [Q&A]
比特球网盘，这个网盘的磁力下载挺强的，下载速度也还行，很多电影，动漫都可以使用磁力直接保存到网盘中，建议集成进来
![image](https://user-images.githubusercontent.com/18629921/205637473-d2fb0224-467d-4817-b382-000d678222b7.png)

**Comment by youtehub** [comment]
比特球这款网盘有大佬在集成吗，一直再用他的离线下载，挺强的，不过貌似api要从网页版去剥离来调用

**Comment by youtehub** [comment]
我这是将以前的留言刷新出来，这是上个月都已经提出了的

**Comment by Gumingjie0312** [comment]
不太可能，太小众了，跑路你都不知道

**Comment by youtehub** [comment]
一般就用来看一下电影之类的，使用 alist  结合 rclone 挂载，plex 用作播放器，跑路也不用担心，因为就只是用来做远程磁盘，视频库，应该不用担心吧

**Reply by nblog** [reply]
我有写python版的登录 下载 上传, 但是不会go, 不了解项目也没办法PR 🤣

**Reply by youtehub** [reply]
可以提供一下api文档吗，看看项目维护人员是否可以帮忙集成一下

**Reply by nblog** [reply]
[bitqiu.py](https://gist.github.com/nblog/b1c0f3ab1b919381318876ae00afcfc3)  重构了 其他功能有空了再更新

**Reply by adminchenyu** [reply]
大佬，有办法挂载比特球云盘吗？

**Comment by youtehub** [comment]
刷新，建议支持一下，有文档

**Comment by dahangda** [comment]
建议支持一下，2024了没跑

**Comment by zeusis666** [comment]
支持，希望能够添加比特球云盘，这是上市公司出的网盘。没那么容易跑路，这么多年都很稳
